store/todos.js

```diff
@@ -1,5 +1,10 @@
 import { uuid } from 'uuidv4';
 export const state = () => ({
+  limit: 9,
+  totalTask: 0,
+  perPage: 9,
```

Comment:
Following prev review, maybe do this instead
```suggestion
  perPage: PER_PAGE,
```

Notice the constant naming.

Reply:
perPage is not a constant but limit is. So i can make a constants.js for Limit

Reply:
arbitary values are problematic in the long run. use constant like `PER_PAGE` as @hhu-wtag bhai suggested as a starting point, even if `perPage` updates later.

Reply:
Resolved